src/pymor/operators/numpy.py

```diff
@@ -531,3 +531,111 @@ def H(self):
             range_id=self.source_id,
             name=self.name + '_adjoint',
         )
+
+
+class MoebiusTransform(NumpyGenericOperator):
+    """A Moebius transform operator.  Maps complex square matrices to complex square matrices.
```

Comment:
Yes, I agree, but it does not seem to fit into the algorithms or tools modules either... Since the Moebius transfrom maps square matrices to square matrices and VectorArrays are (ab)used as matrices in other places in pyMOR, I went with an `Operator` implementation that checks that `len(U) == U.dim` in `MoebiusTransform.apply(U)`.

Does anyone have a better idea?

Reply:
Maybe it fits in the `models` module?
Also, maybe it's better to think of the Moebius transform just as a rational function, without `apply` and `apply_inverse` methods.

Reply:
> Maybe it fits in the `models` module?

That would be possible. It would actually be nice, if it could take and return `Operators`. Do you think there is merit in implementing other operator -> operator mappings in a similar way? 

> Also, maybe it's better to think of the Moebius transform just as a rational function, without `apply` and `apply_inverse` methods.

I like the fact that they are objects, because it allows for different constructors and the group structure is reflected by the `matmul` method, so that multiple transforms can be combined into one. Could this be achieved with just a function?

Reply:
I agree, I meant implementing `MoebiusTransform` as a class, but without `Operator` semantics (I guess the `coefficients` attribute and the `__matmul__` method would be enough). This is because I don't see what you gain by treating it as a mapping from `Operators` to `Operators`, besides a way to write the transformed `A`. You would still transform `B`, `C`, and `D` by hand, so why not also `A`.

Reply:
I intended to make the implementation as general as possible such that it could be used elsewhere in the future. I totally agree, that this is not needed in the `LTIModel` and so I am also happy to move a scalar version to the models module.

Reply:
I just starting doing that, but I realized that some kind of `apply` is still needed in case one wants to actually apply the Moebius transform to some frequencies. I'm fine with only having a scalar implementation though, so it does not necessarily need to work on `VectorArrays`. Should I still call the method `apply` or is it better to chose a different approach?

Reply:
What do you mean by an `apply` method here? A method that takes `s` and returns `(a*s+b)/(c*s+d)`? You could use `__call__` to do that if you'd like.

Reply:
okay, great. I have moved it over to `models.transforms` and inherited from `Model`. Let me know if that is what you had in mind!